pwiz_tools/Shared/CommonUtil/SystemUtil/PathEx.cs

```diff
@@ -376,6 +377,38 @@ public static string GetRandomFileName()
             return result;
         }
 
+        /// <summary>
+        /// Like PathEx.GetRandomFileName(), but tries to ensure that no unicode characters are in the name
+        /// </summary>
+        /// <returns>a random filename containing only ASCII characters</returns>
+        public static string GetNonUnicodeRandomFileName()

```

Comment:
This function seems odd to me. Couldn't we just give GetRandomFileName() an optional argument like `bool addUnicodeTestString` or something and then this function could call that with `false`, or just whatever is going to use this function would directly call GetRandomFileName like that?

Reply:
hah, the whole thing turns out to be cruft, deleting it